runtime/docker/docker.go

```diff
@@ -1003,3 +1003,16 @@ func (d *DockerRuntime) containerPid(ctx context.Context, cID string) (int, erro
 	}
 	return inspect.State.Pid, nil
 }
+
+// GetContainerHealth returns true is the container is reported as being healthy, false otherwise.
+func (d *DockerRuntime) GetContainerHealth(ctx context.Context, cID string) (bool, error) {
```

Comment:
I think `IsHealthy` is a better func name here.

Reply:
Done